tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/util/NginxUtil.java

```diff
@@ -97,6 +139,25 @@ public static void deployNginx(
                     .withMountPath("/usr/share/nginx/html/" + fileName)
                     .build());
         }
+        if (withHttps) {
+            volumes.add(new VolumeBuilder().withName("nginx-certificates")
+                    .withNewSecret()
+                    .withSecretName("nginx-certificates")
+                    .endSecret()
+                    .build());
+            volumeMounts.add(new VolumeMountBuilder().withName("nginx-certificates")
+                    .withMountPath("/etc/nginx/certificates")
+                    .build());
+            volumes.add(new VolumeBuilder().withName("nginx-https-conf")
+                    .withNewHostPath()
```

Comment:
Nice! didn't know about this hack

Reply:
Maybe a ConfigMap would have been more in line with the other setups we have. Not sure why I went with the "copy files into the host container" approach in the first place. The lifecycle of this util is anyway not perfect. It relies on the test to delete the namespace to cleanup everything.